vscode-extension/src/aiConfigEditor.ts

```diff
@@ -110,6 +96,38 @@ export class AIConfigEditorProvider implements vscode.CustomTextEditorProvider {
       });
     }
 
+    // Update webview immediately so we unblock the render; server init will happen in the background.
+    updateWebview();
+
+    // Start the AIConfig editor server process. Don't await at the top level here since that blocks the
+    // webview render (which happens only when resolveCustomTextEditor returns)
+    this.startEditorServer(document).then(async (editorServer) => {
+      editorServer = editorServer;
+
+      this.aiconfigEditorManager.addEditor(

```

Comment:
This seems fine, but might be worth having an update when an editor server is created for aiConfigManager to update its state. Because technically aiConfigManager should keep track of all open editors irrespective of the server.

Not blocking for this diff since this is the same functionality as before